tests/utils/helpers.py

```diff
@@ -25,6 +25,27 @@ def hex_string_to_bytes_array(h: str):
     return [int(b, 16) for b in wrap(h, 2)]
 
 
+def hex_string_to_uint256(h: str):
```

Comment:
there is already a `from tests.utils.uint256 import int_to_uint256` so:
- probably put this there
- use it once you've got a int